packages/p2p/src/components/buy-sell/sort-dropdown.jsx

```diff
@@ -9,13 +9,18 @@ import 'Components/buy-sell/sort-dropdown.scss';
 const SortDropdown = () => {
     const { buy_sell_store } = useStores();
 
+    const sort_list = [
```

Comment:
the language change doesnt reflect in store or even in constant file. so i need to create a local variable instead